lib/index.js

```diff
@@ -4,58 +4,58 @@ var fs = require('fs'),
     ngHtml2Js = require('ng-html2js'),
     transformify = require('transformify');
 
-function isExtension (file, extension) {
-  return new RegExp('\\.' + extension + '$').test(file);
+function isExtension(file, extension) {
+    return new RegExp('\\.' + extension + '$').test(file);
```

Comment:
could you re-indent the code back to 2 spaces, to keep the project consistent :)